front/src/Data.elm

```diff
@@ -373,6 +410,174 @@ lilletBlanc =
     { name = "Lillet blanc", t = Spirit }
 
 
+irishWhiskey : Material
+irishWhiskey =
+    { name = "Irish whiskey", t = Spirit }
+
+
+oldTomGin : Material
+oldTomGin =
+    { name = "Old Tom Gin", t = Spirit }
```

Comment:
How, tom, *how* could you not use old tom, whilest you are called tom, and this website’s hostname includes *old*.

Reply:
Haha, I don't think I've even had old tom gin! Will definitely try it out once things settle down and I'll be older then too